utils/build-script-impl

```diff
@@ -1193,6 +1297,46 @@ function build_directory_bin() {
     fi
 }
 
+function get_host_install_destdir() {
```

Comment:
Could you make this function echo the value instead of setting a global?  It would be more in line with other functions in this file that are callable via `result=$(foobar)` syntax.